scripts/lib/format-converter.ts

```diff
@@ -224,3 +230,93 @@ export function googleSheetToFlatData(
   }
   return records;
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type StringKeyed = { [index: string]: any };
+
+export function nestedToFlat(input: StringKeyed): StringKeyed {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const output: { [index: string]: any } = {};
```

Comment:
value? or fieldValue?

Reply:
Sorry, I don't understand what this comment is referring to.

Reply:
oh, sorry, I probably saved it to the wrong line.

for (const [fieldName, field]) 

this is referring to renaming 'field'. Specifically due to believing that if fieldName exists (since it's in input's entries) it seems weird to check if the field itself is undefined (when really it's checking if the value of the field doesn't exist here, and is checking the contents of the named field).


Does that make sense?